File: DotNet/Apps/ConfMgrGUI/OpsTrackingModel/BdtaSeCptyLkup.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OpsTrackingModel
{
    [System.Serializable]
    public class BdtaSeCptyLkup
    {
        private string _cptySn = null;


        public BdtaSeCptyLkup()
        {
            this._cptySn = null;
        }
        
        public string CptySn
        {
            get { return _cptySn; }
            set { _cptySn = value; }
        }
    }
}

File: DotNet/Apps/ConfMgrGUI/OpsTrackingModel/src/OpsTrackingModel/BdtaSeCptyLkup.java
package OpsTrackingModel;

import java.io.Serializable;

//import com.gigaspaces.annotation.pojo.SpaceClass;
//import com.gigaspaces.annotation.pojo.SpaceId;
//import com.gigaspaces.annotation.pojo.SpaceRouting;

//@SpaceClass
public class BdtaSeCptyLkup implements Serializable{

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	
	private String _cptySn = null;
	
  //  @SpaceId    
 //   @SpaceRouting
	public String get_cptySn() {
		return _cptySn;
	}
	public void set_cptySn(String sn) {
		_cptySn = sn;
	}
}
